virttest/qemu_vm.py

```diff
@@ -993,13 +992,12 @@ def add_memorys(devices, params):
             output = re.findall(r",memory-backend=mem-([\w|-]+)", machine_cmd)
             if output:
                 name = output[0]
-                backend_options = {}
+                backend_options = params.object_params(name)
                 backend_options["size"] = "%sM" % params["mem"]
                 if params.get("hugepage_path"):
                     backend_options["backend"] = "memory-backend-file"
                     backend_options["mem-path"] = params["hugepage_path"]
```

Comment:
Hold on... After the change, are these lines necessary?

Reply:
After testing, It needs to set the specified mem-path globally.
So it is necessary.

Reply:
@huangyum hi, do you agree with me?

Reply:
Yes, I'd like to keep them. If we remove these lines, we will have to add params such as 'mem-path_machine_mem' for all hugepage cases.